app/jobs/cache_analytics_job.rb

```diff
@@ -37,56 +47,63 @@ def perform(*_args)
+    counts.concat(monitoree_counts_by_risk_factor(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_exposure_country(analytic_id, monitorees))
 
     # Active and overall counts for date of last exposure
-    counts.concat(monitoree_counts_by_last_exposure_date(analytic_id, monitorees, true))
-    counts.concat(monitoree_counts_by_last_exposure_date(analytic_id, monitorees, false))
-    counts.concat(monitoree_counts_by_last_exposure_week(analytic_id, monitorees, true))
-    counts.concat(monitoree_counts_by_last_exposure_week(analytic_id, monitorees, false))
-    counts.concat(monitoree_counts_by_last_exposure_month(analytic_id, monitorees, true))
-    counts.concat(monitoree_counts_by_last_exposure_month(analytic_id, monitorees, false))
-
+    counts.concat(monitoree_counts_by_last_exposure_date(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_last_exposure_week(analytic_id, monitorees))
+    counts.concat(monitoree_counts_by_last_exposure_month(analytic_id, monitorees))
     counts
   end
 
-  # Total monitoree counts
-  def self.monitoree_counts_by_total(analytic_id, monitorees, active_monitoring)
-    monitorees.monitoring_active(active_monitoring)
-              .group(:exposure_risk_assessment)
-              .order(:exposure_risk_assessment)
+  # Monitoree counts by age group
+  def self.monitoree_counts_by_age_group(analytic_id, monitorees)
+    counts = []
+    # Some jurisdictions are using `1-1-1900` as a "fake birthdate" where data might be invalid or unknown
+    # This can skew the `>=80` analytics data, so we collect the count of monitoree's over 110 years old
+    # And inform the user that that number is bundled in with `>=80`
+    # The client will perform the logic to combine the "FAKE_BIRTHDATE" in with `>=80`
+    age_groups = <<-SQL
+      CASE
+        WHEN TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) < 20 THEN '0-19'
+        WHEN TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) >= 20 AND TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) < 30 THEN '20-29'
+        WHEN TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) >= 30 AND TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) < 40 THEN '30-39'
+        WHEN TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) >= 40 AND TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) < 50 THEN '40-49'
+        WHEN TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) >= 50 AND TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) < 60 THEN '50-59'
+        WHEN TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) >= 60 AND TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) < 70 THEN '60-69'
+        WHEN TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) >= 70 AND TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) < 80 THEN '70-79'
+        WHEN TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) >= 80 AND TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) < 110 THEN '>=80'
+        WHEN TIMESTAMPDIFF(YEAR, date_of_birth, CURDATE()) >= 110 THEN 'FAKE_BIRTHDATE'
```

Comment:
I'm wondering if this should also include any negative numbers too

Reply:
I was unable to enroll any monitorees with a date of birth in the future. Does the API provide checks for that too?